change/@ni-nimble-blazor-20bd54c7-238d-4715-936b-11efce400236.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Both 1 and 2 seem bad. It seems like some bindings and event callbacks aren't getting triggered correctly right now for InteractiveWebAssembly rendering (at least in this example app), I'll investigate more.

> All three of these are also true on the existing example app

 I don't see that - 1 and 2 work for me currently at https://nimble.ni.dev/storybook/blazor-client-app/wwwroot/ .

Reply:
1 and 2 are fixed now (updates to the JS initialization scripts).

Side note, it looks like the problem was specific to Blazor WASM Standalone projects, so adding more `InteractiveWebAssembly` render mode tests to `NimbleBlazor.Tests.Acceptance` wouldn't catch the issue either (since they're launched via a Blazor Web App, a different set of startup events/callbacks happen).

Reply:
>I don't see that - 1 and 2 work for me currently at https://nimble.ni.dev/storybook/blazor-client-app/wwwroot/ .

Hmm, I can't reproduce those issues in the existing example app now either. I must have got my browser tabs mixed up.

Thanks for fixing the issues! I validated 1 and 2 no longer reproduce in this latest build: https://60e89457a987cf003efc0a5b-bturyhpprk.chromatic.com/blazor-client-app/wwwroot

Reply:
Filed #2461 for the 3rd issue. Didn't immediately see anything wrong in the demo Razor page (like using AutoFocus unexpectedly).